evaluation/benchmarks/analytics-mts/input/setup.sh

```diff
@@ -13,13 +13,26 @@ if [[ "$1" == "-c" ]]; then
     exit
 fi
 
-if [ ! -f ./in.csv ]; then
-  # yesterday=$(date --date='1 days ago' +'%y-%m-%d')
-  # curl https://www.balab.aueb.gr/~dds/oasa-$yesterday.bz2 |
-  curl -sf 'https://www.balab.aueb.gr/~dds/oasa-2021-01-08.bz2' | bzip2 -d > in.csv
-  if [ $? -ne 0 ]; then
-    echo "oasa-2021-01-08.bz2 / bzip2 not available, contact the pash authors"
-    exit 1
+setup_dataset() {
+  if [ ! -f ./in.csv ]; then
+    # yesterday=$(date --date='1 days ago' +'%y-%m-%d')
+    # curl https://www.balab.aueb.gr/~dds/oasa-$yesterday.bz2 |
+    curl -sf 'https://www.balab.aueb.gr/~dds/oasa-2021-01-08.bz2' | bzip2 -d > in.csv
+    if [ $? -ne 0 ]; then
+      echo "oasa-2021-01-08.bz2 / bzip2 not available, contact the pash authors"
+      exit 1
+    fi
+    "$PASH_TOP/scripts/append_nl_if_not.sh"  in.csv
+    len=$(cat in.csv | wc -l)
+    half_size=$(( $len / 4 ))
+    head -n $half_size in.csv > in_small.csv
   fi
-  "$PASH_TOP/scripts/append_nl_if_not.sh"  in.csv
-fi
+}
+
+source_var() {
```

Comment:
So what is the plan for using it? we need to `source setup.sh` and then run the actual test?

Reply:
I think that is OK! My confusion was resolved when I saw the `download_...` function in `utils.sh`